### PR TITLE
Hide Attendees Label if no Attendees Present in View Event Modal 

### LINKED ISSUE #65: Hide Attendees Label if no Attendees Present in View Event Modal
Hide the attendees text in view event modal if the attendees array is empty.

Screenshot 2020-05-18 at 1 39 49 AM

### PR DESCRIPTION
**Pull request by coderrsid**
# Fixes #65

### CODE REVIEW

File: app/modals/ViewEvent/Content/index.js [hunk 1]
```diff
@@ -93,10 +93,14 @@ const EventContent = memo(({ event, onClose }) => {
           <strong>Organiser:</strong>
         </p>
         <AttendeeItem className="mb-4" invite={organiserInvite} event={event} />
-        <p>
-          <strong>Attendees:</strong>
-        </p>
-        <AttendeesList className="mb-4" event={event} />
+        {event.invites && (
```
Comment (0xnmn) [inline app/modals/ViewEvent/Content/index.js]: !![] is true. What is the array is empty?

**Review by 0xnmn — CHANGES_REQUESTED** [review]

File: app/modals/ViewEvent/Content/index.js (hunk 1 above)
Comment (coderrsid) [reply app/modals/ViewEvent/Content/index.js]: So can i check the array & length instead?
like: ([].isArray([]) && [].length)

Comment (0xnmn) [reply app/modals/ViewEvent/Content/index.js]: `!!event.invites?.length`

Comment (coderrsid) [reply app/modals/ViewEvent/Content/index.js]: Yeah exactly.. so that'll work right?

Comment (0xnmn) [reply app/modals/ViewEvent/Content/index.js]: yes

Comment (coderrsid) [reply app/modals/ViewEvent/Content/index.js]: Done!

**Review by 0xnmn — APPROVED** [review]

**Pull request merged**